plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/actionworkers/KubernetesClusterResourceModifierActionWorker.java

```diff
@@ -307,7 +307,6 @@ protected void startKubernetesVM(final UserVm vm) throws ManagementServerExcepti
             Field f = startVm.getClass().getDeclaredField("id");
             f.setAccessible(true);
             f.set(startVm, vm.getId());
-            resizeNodeVolume(vm);
```

Comment:
Would that cause an issue (i.e. not resizing node on start?)

Reply:
the resizing happens before this is invoked, having this effectively meant we we attempting to resize twice. Removed it because it is redundant

Reply:
Checked, resize call is present in `provisionKubernetesClusterNodeVms` at line 331